### PR TITLE
feat: add fixture factory for item and member

### PR DESCRIPTION
**Pull request by pyphilia**
The factories provide random fixtures for member and item.
These are only used in the backend for now, I think it should be fully adopted in the backend before moving to the frontend because the frontend tends to do everything at the same time (one json schema) while the backend is really modularized (creates each entity separately). 

Next step would be to include a factory for favorite. The idea would be to provide `{ ...itemProperties, favorite: [{memberId}] }` eventually.

- [ ] write tests

### CODE REVIEW

**Review by ReidyT — APPROVED** [review]
Really nice 👍 !

File: src/services/items/itemFactory.ts [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+          },
+        },
+      };
+  }
+};
+
+faker.helpers.arrayElement([
+  { lang: faker.helpers.arrayElement(['en', 'fr', 'de']) },
+  {},
+]);
```
Comment (spaenleh) [inline src/services/items/itemFactory.ts:122]: Remove ?

File: src/services/items/itemFactory.ts [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+  const updatedAt = faker.date.anytime();
+
+  const path =
+    (i.parentItem ? i.parentItem.path + '.' : '') + buildPathFromIds(id);
+
+  return {
+    id,
+    name: faker.person.fullName(),
+    description: faker.lorem.text(),
+    createdAt: adaptDate(createdAt, options) as any, // TODO need help: depend on the generic type
```
Comment (spaenleh) [inline src/services/items/itemFactory.ts:168]: The idea here would be for the function to get a parameter that tels how the dates should be formatted. For example: `dateFormat: 'string'` or `dateFormat: 'date'`. Based on this param, create the dates in the right format. 
The return type of the factory function should be either the one with string or with Date. The return type can be narrowed by linking the value of the argument that is passed at runtime with the output type by the means of an intermediary generic type. 

I will try to do that on my own, I let you try it too, and we can check together after.

File: src/services/members/memberFactory.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  id: faker.string.uuid(),
+  name: faker.person.fullName(),
+  email: faker.internet.email(),
+  createdAt: faker.date.anytime().toISOString(),
+  updatedAt: faker.date.anytime().toISOString(),
+  extra: faker.helpers.arrayElement([
+    { lang: faker.helpers.arrayElement(['en', 'fr', 'de']) },
+    {},
+  ]),
+  type: MemberType.Individual, // todo
```
Comment (spaenleh) [inline src/services/members/memberFactory.ts:16]: Can you explicit the TODO here ?

**Pull request merged**